css/gallery.css

```diff
@@ -51,7 +51,156 @@
 
 .gallery__items :nth-child(2n) {
   margin-right: 0;
+}     
+.wiggle {
+  display:block;
+  margin:0 auto;
+  width:150px;
+  height:150px;
+  box-shadow:8px 8px 1px 0 rgba(0,0,0,.15);
 }
+ 
+.wiggle:hover {
+  -webkit-animation:none;
+}
+ 
+.short-animate {
+  -webkit-transition:.5s ease-in-out;
+  -moz-transition:.5s ease-in-out;
+  -ms-transition:.5s ease-in-out;
+  -o-transition:.5s ease-in-out;
+  transition:.5s ease-in-out;
+}
+ 
+.long-animate {
+  -webkit-transition: .5s .5s ease-in-out;
+  -moz-transition: .5s .5s ease-in-out;
+  -ms-transition: .5s .5s ease-in-out;
+  -o-transition:.5s .5s ease-in-out;
+  transition:.5s .5s ease-in-out;
+}
+ 
+html,body {
+  height:100%;
+  min-height:100%;
+}
+ 
+.lightbox {
+  position:fixed;
+  top:-100%;
+  bottom:100%;
+  left:0;
+  right:0;
+  background:#003c8f;
+  z-index:501;
+  opacity:0;
+}
+ 
+.lightbox img {
+  position:absolute;
+  margin:auto;
+  top:0;
+  left:0;
+  right:0;
+  bottom:0;
+  max-width:0%;
+  max-height:0%;
+}
+ 
+.lightbox-controls {
+  position:fixed;
+  height:70px;
+  width:70px;
+  top:-70px;
+  right:0;
+  z-index:502;
+  background:rgba(0,0,0,.1);
+}
+ 
+.close-lightbox {
+  display:block;
+  position:absolute;
+  overflow:hidden;
+  height:50px;
+  width:50px;
+  text-indent:-5000px;
+  right:10px;
+  top:10px;
+  -webkit-transform:rotate(45deg);
+  -moz-transform:rotate(45deg);
+  -ms-transform:rotate(45deg);
+  -o-transform:rotate(45deg);
+  transform:rotate(45deg);
+}
+ 
+.close-lightbox:before {
+  content:'';
+  display:block;
+  position:absolute;
+  height:0px;
+  width:3px;
+  left:24px;
+  top:0;
+  background:white;
+  border-radius:2px;
+  -webkit-transition: .5s .5s ease-in-out;
+  -moz-transition: .5s .5s ease-in-out;
+  -ms-transition: .5s .5s ease-in-out;
+  -o-transition:.5s .5s ease-in-out;
+  transition:.5s .5s ease-in-out;
+}
+ 
+.close-lightbox:after {
+  content:'';
+  display:block;
+  position:absolute;
+  width:0px;
+  height:3px;
+  top:24px;
+  left:0;
+  background:white;
+  border-radius:2px;
+  -webkit-transition: .5s 1s ease-in-out;
+  -moz-transition: .5s 1s ease-in-out;
+  -ms-transition: .5s 1s ease-in-out;
+  -o-transition:.5s 1s ease-in-out;
+  transition:.5s 1s ease-in-out;
+}
+ 
+.lightbox:target {
+  top:0%;
+  bottom:0%;
+  opacity:1;
+}
+ 
+.lightbox:target img {
+  max-width: 90%;
+  max-height: 90%;
+}
+ 
+.lightbox:target ~ .lightbox-controls {
+  top:0px;
+}
+ 
+.lightbox:target ~ .lightbox-controls .close-lightbox:after {
+  width:50px;
+}
+ 
+.lightbox:target ~ .lightbox-controls .close-lightbox:before {
+  height:50px;
+}
+ 
+@-webkit-keyframes wiggle {
```

Comment:
Wiggle is a tag? what do you do in this line?

Reply:
Hace parte de la animación del svg de cerrado.